params/config.go

```diff
@@ -265,7 +265,7 @@ var (
 		BrunoBlock:          big.NewInt(13082000),
 
 		// TODO modify blockNumber, make sure the blockNumber is not an integer multiple of 200 (epoch number)
-		// TODO Caution !!! it should be very careful !!!
+		// Caution !!! it should be very careful !!!
```

Comment:
Keep TODO in this PR, once we have the upgrade plan, we can propose another PR to modify this upgrade height.

Reply:
Fixed